### PR TITLE
Add Tegunov-style wiener filter to applyctf & allow ctffind to work on low-mag images

### PR DESCRIPTION
**Pull request by jojoelfe**
This pull request does two things:

- It allows the applyctf program to apply the Tegunov-style wiener as implemented in the image class
- It adds a command-line flag to ctffind that allows setting the minimum resolution to arbitrary values

The use case for this at the moment is to apply the filter to low-magnification images with ~4 nm pixel-size.

### CODE REVIEW

**Comment by jojoelfe** [comment]
One thing to point out is that the old-school-input from ctffind does not check the minimum resolution input, so the command-line flag might not be absolutely necessary.

File: src/core/image.h [hunk 1]
```diff
@@ -134,7 +134,7 @@ class Image {
 	void WeightBySSNR(Image &ctf_image, float molecular_mass_kDa, float pixel_size, Curve &SSNR, Image &projection_image, bool weight_particle_image, bool weight_projection_image);
 	void OptimalFilterSSNR(Curve &SSNR);
 	void OptimalFilterFSC(Curve &FSC);
-	void OptimalFilterWarp(CTF ctf, float pixel_size_in_angstroms, float ssnr_falloff_fudge_factor = 1.0, float ssnr_scale_fudge_factor = 1.0);
+	void OptimalFilterWarp(CTF ctf, float pixel_size_in_angstroms, float ssnr_falloff_frequency = 100.0, float ssnr_falloff_fudge_factor = 1.0, float ssnr_scale_fudge_factor = 1.0, float hp_frequency=200.0);
```
Comment (bHimes) [inline src/core/image.h:137]: So the changes to the image class are intended to allow flexibility in the parameters used in the Wiener filter, right? Have you found any other than the default that are useful? I know I made them editable in emClarity too, but never use anything but the default.

**Review by bHimes — COMMENTED** [review]
Per your comment on ctffind: If you can get around this issue of the low-resolution limit by using the existing old-school-input flag, is there any reason not to just do that? As long as the estimation still works okay, I would be inclined to suggest we avoid changes to ctffind. If decide they are needed, let's put the ctffind changes in their own PR, as Alexis will specifically need to review.

Apply CTF looks good, consider those approved.

Image Class - See inline comment.

File: src/core/image.h (hunk 1 above)
Comment (jojoelfe) [reply src/core/image.h:137]: Yes, for low magnification images. If the pixel size is 40A, a falloff frequency of 100A is not very useful. The high-pass filter does not seem to change things too much, but at that point it seems weird to not have it adjustable.

**Comment by jojoelfe** [comment]
Ok, just tried to force push the branch with the ctffind commits removed, but this seems to conflict with the base branch. Is split_j the branch you want me to use? I'll try to rebase and if that doesn't work I will close and reopen.

**Comment by jojoelfe** [comment]
Ok, rebase onto split_j and things seems ok now

**Pull request merged**